modules/hcp-nodepool-hc.adoc

```diff
@@ -33,15 +33,15 @@ spec:
     image: registry.<dns.base.domain.name>:5000/openshift/release-images:4.x.y-x86_64 \// <5>
   replicas: 0
 status:
-  replicas: 0 // <6>
+  replicas: 2 // <6>
```

Comment:
the `// <6>` should go into the above one in the `spec` section not in the `status` one. Status should show the same number as spec.